server/stream.go

```diff
@@ -1347,6 +1387,11 @@ func (mset *stream) setSourceConsumer(sname string, seq uint64) {
 
 	b, _ := json.Marshal(req)
 	subject = fmt.Sprintf(JSApiDurableCreateT, sname, durable)
+	if ext != nil {
+		subject = strings.Replace(subject, JSApiPrefix, ext.ApiPrefix, 1)
```

Comment:
This code is repeated 4 times, so it could have been worth a function. Even the one with either `$JS.M` or `$JS.S` could have been made a function.

Reply:
Agree and post GA lots of little cleanup opportunities like that.